RCTDeviceUUID.m

```diff
@@ -17,13 +17,7 @@ @implementation RCTDeviceUUID
 
 RCT_EXPORT_METHOD(getUUID:(RCTResponseSenderBlock)callback)
 {
-  NSUUID *deviceId;
-
-#if TARGET_IPHONE_SIMULATOR
-  deviceId = [[NSUUID alloc]initWithUUIDString:@"UUID-STRING-VALUE"];
-#else
-  deviceId = [UIDevice currentDevice].identifierForVendor;
-#endif
+  NSUUID *deviceId =  [UIDevice currentDevice].identifierForVendor;
```

Comment:
Worked on the simulator and TestFlight.
![image](https://cloud.githubusercontent.com/assets/6221799/9328352/6d4dcfc0-457e-11e5-85de-68514c586536.png)